phpBB/includes/functions_mcp.php

```diff
@@ -575,6 +553,54 @@ function phpbb_mcp_sorting($mode, &$sort_days, &$sort_key, &$sort_dir, &$sort_by
 			break;
 	}
 
+
```

Comment:
Please remove this extra new line. This will also resolve the travis build fail.